### DISCUSSION TITLE
Márgenes en Grid

### DISCUSSION THREAD

**Discussion by Jose-Galicia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#340] [General]
Hola buenas tardes, he estado trabajando con la actividad de CSS y use un grid para realizar columnas, el problema es que al hacer esto me da margenes a los costados y no he podido eliminarlos, ya probe varias cosas y no me ha resultado nada, alguien tiene una idea de como quitar esos margenes?.
![Captura](https://user-images.githubusercontent.com/99226240/157742179-b9cb0c79-de89-41a4-b5b1-74ec6cc07639.PNG)

**Comment by DanyVeneno** [comment]
Que tal José te dejo este link no se que estes usando para maquetar este es un curso de css grid
1-https://www.youtube.com/watch?v=gCox2CqYvto&list=PLvq-jIkSeTUY628cyd9LVbXSXi2xG9mUl&ab_channel=jonmircha
2-Otra referencia / https://css-tricks.com/guides/
pueden ser varias cosas pero seguro estos dos link te van a ayudar a solucionarlo
Saludos  buena tarde

**Reply by Jose-Galicia** [reply]
Muchas gracias, voy a checarlas.

**Comment by DanyVeneno** [comment]
Y te dejo otro enlace de una app para escritorioen la cual puedes saber todos los colores de tu pantalla en sus diferentes valores
https://annystudio.com/software/colorpicker/

**Comment by carlos-gmx3** [comment]
Parece ser cosa de que ajustes el valor de grid-gap el cual define el espacio entre celdas de un grid

**Comment by Jimdrer** [comment]
> Hola buenas tardes, he estado trabajando con la actividad de CSS y use un grid para realizar columnas, el problema es que al hacer esto me da margenes a los costados y no he podido eliminarlos, ya probe varias cosas y no me ha resultado nada, alguien tiene una idea de como quitar esos margenes?.
> ![Captura](https://user-images.githubusercontent.com/99226240/157742179-b9cb0c79-de89-41a4-b5b1-74ec6cc07639.PNG)
> 
> 
> 
> 
> 

En tu css agrega ésto al principio para que todo tenga un mejor ajuste, espero te sirva de ayuda 😀
 
 ​*​{ 
 ​    ​box-sizing​:​ border-box; 
 ​    ​margin​:​ ​0​; 
 ​    ​padding​:​ ​0​; 
 ​}

**Reply by Jose-Galicia** [reply]
Ya están definidos de esa manera y aun asi no puedo haha

**Comment by OmarVenturaP** [comment]
Como comenta el compañero @Jimdrer, hay que colocarle los parámetros en 0, xk cada navegador tiene sus propios parámetros, ya con esa configuración lo dejas en blanco para poder aplicarle los estilos que desees.

**Reply by Jose-Galicia** [reply]
Si, ya esta definido de esa manera, pero aun los mantiene.
